Libplanet/Net/Protocols/KademliaProtocol.cs

```diff
@@ -180,7 +185,6 @@ public class KademliaProtocol : IProtocol
             {
                 _logger.Debug(
                     $"Timeout occurred during {nameof(AddPeersAsync)}() after {timeout}.");
-                throw;
```

Comment:
Then now, just logging and ignore `TimeoutException`?

Reply:
Right.